### ISSUE TITLE
Any chance you could provide any more information on this project?

### ISSUE THREAD

**Issue by datadrian** [maitreyastark/Raspberry-Pi-Autonomous-Car#1]
how is the training data obtained? do you have a training mode? I'm working on a similar project to have a cart follow me on the beach.

Thanks!

**Comment by maitreyastark** [comment]
Well, i have used a trained model.... If you can get caffemodel installed on your machine you have a lot of tutorials available for creating your own neural network. Begin with sentdex's (youtube) video on binary cat dog image classification and progress towarda a multiclass model to recognise your face....

**Comment by datadrian** [comment]
what was the model trained on?

Adrian Stucker
513.341.LENS

**Comment by maitreyastark** [comment]
I used a pre trained model.... it was trained using CNN over 14 classes.

**Comment by datadrian** [comment]
understood, thank you! I would love to see any pictures or video of your
tests.

Adrian Stucker
513.341.LENS
